activestorage/lib/active_storage/attached/one.rb

```diff
@@ -62,28 +63,24 @@ def detach
       end
     end
 
+    ##
+    # :method: purge
+    #
     # Directly purges the attachment (i.e. destroys the blob and
     # attachment and deletes the file on the service).
-    def purge
-      if attached?
-        attachment.purge
-        write_attachment nil
-        reset_changes
-      end
-    end
 
+    ##
+    # :method: :purge_later
```

Comment:
Was this necessary?

Reply:
the `:method:` is needed in order to generate correctly the RDOC.
The semicolon before the name is a typo: I've removed it and pushed the fix.